plugins/modules/win_format.ps1

```diff
@@ -19,22 +19,30 @@ $spec = @{
         file_system = @{ type = "str"; choices = "ntfs", "refs", "exfat", "fat32", "fat" }
         allocation_unit_size = @{ type = "int" }
         large_frs = @{ type = "bool" }
-        full = @{ type = "bool"; default = $false }
-        compress = @{ type = "bool" }
+        full_format = @{ type = "bool"; default = $false }
```

Comment:
We can't change the existing options, if you wish to create ones that are a bit more explanatory then you should add an `alias` instead.

```suggestion
        full = @{ type = "bool"; default = $false; aliases = @(,'full_format') }
```